x-pack/plugins/lens/public/visualizations/datatable/components/table_basic.test.tsx

```diff
@@ -141,9 +141,9 @@ describe('DatatableComponent', () => {
     expect(screen.getByLabelText('My fanci metric chart')).toBeInTheDocument();
     expect(screen.getByRole('row')).toBeInTheDocument();
     expect(screen.queryAllByRole('gridcell').map((cell) => cell.textContent)).toEqual([
-      'shoes- a, column 1, row 1',
-      '1588024800000- b, column 2, row 1',
-      '3- c, column 3, row 1',
```

Comment:
I'm wondering why this has changed

Reply:
Follow up note: if you still wish to assert on cell col/row placement in EuiDataGrid, please use the `data-gridcell-column-index` and `data-gridcell-visible-row-index` attributes.